jobs/travelnest_edinburgh_20180604.html

```diff
@@ -0,0 +1,54 @@
+---
+title: Software Development Engineers (All levels)
+company: TravelNest
+url: https://travelnest.breezy.hr/p/8f72c74d67bd01-software-development-engineers-all-levels
+location: Edinburgh,  United Kingdom
+contract: permanent
+contact:
+    name: Kim Watson
+    email: kim.watson@travelnest.com
+    phone: +447398971079
```

Comment:
the `+` here doesn't seem to be rendering. Looks to be working on with similar markup on http://pythonjobs.github.io/jobs/headforward-python.html 
from source https://github.com/pythonjobs/jobs/blob/master/jobs/headforward-python.html

Reply:
Hi Laurence 

I think what’s happening here is the YAML parser is treating the whole thing as a number, so ignoring the + when outputting.  

In the other example, the number has spaces in which may be enough to make the parser revert to string

If you add spaces, or put “” marks around the whole number, I’m guessing that should fix it

Thanks

Steve

Reply:
thanks for the head up, 1f65967 should fix